### DISCUSSION TITLE
cssBundleHref is undefined

### DISCUSSION THREAD

**Discussion by WreewanMorhee** [remix-run/remix#6803] [Q&A] [answered]
does anyone encounter this problem ?

my cssBundleHref is undefined

so when I use Vanilla Extract, I have className but without any style

I think the reason is that my cssBundleHref is undefined cause this bug

Anyone encounter this either ? 

<img width="609" alt="Screen Shot 2023-07-10 at 6 14 21 PM" src="https://github.com/remix-run/remix/assets/33398071/ad7b01e6-19f0-404f-b563-f72b64e38276">

I get undefined here in root.tsx

here is version I use
<img width="346" alt="Screen Shot 2023-07-11 at 1 54 26 PM" src="https://github.com/remix-run/remix/assets/33398071/683461a5-74bf-4474-8786-5a5a3d2d01ac">

**Comment by MichaelDeBoey** [comment]
CC/ @markdalgleish

**Comment by xHomu** [comment, marked as answer]
You probably want all the remix-run/* bundles to be the same version.

**Reply by markdalgleish** [reply]
That's right. You should always keep your Remix packages on the same version since they're developed, tested and published as a set.

**Reply by WreewanMorhee** [reply]
yes
resolved 

thank u sir !

**Comment by ersouravbansal** [comment]
How we can add css bundle in style tag in head. Currently it is coming in link tag.

export const links: LinksFunction = () => [
  ...(cssBundleHref
    ? [{ rel: "stylesheet", href: cssBundleHref }]
    : []),
  {
    rel: "preconnect",
    href: "https://fonts.googleapis.com",
    crossOrigin: "anonymous",
  },
  {
    rel: "preconnect",
    href: "https://fonts.gstatic.com",
    crossOrigin: "anonymous",
  },
  {
    rel: "stylesheet",
    href: styles
  },
  ];
